sap-abap/services/cloudwatch/README.md

```diff
@@ -0,0 +1,106 @@
+* Running the tests might result in charges to your AWS account.
+* We recommend that you grant your code least privilege. At most, grant only the minimum permissions required to perform the task. For more information, see [Grant least privilege](https://docs.aws.amazon.com/IAM/latest/UserGuide/best-practices.html#grant-least-privilege).
+* This code is not tested in every AWS Region. For more information, see [AWS Regional Services](https://aws.amazon.com/about-aws/global-infrastructure/regional-product-services).
+
+<!--custom.important.start-->
+<!--custom.important.end-->
+
+## Code examples
+
+### Prerequisites
+
+For prerequisites, see the [README](../../README.md#Prerequisites) in the `sap-abap` folder.
+
+
+<!--custom.prerequisites.start-->
+<!--custom.prerequisites.end-->
+
+### Single actions
+
+Code excerpts that show you how to call individual service functions.
+
+- [Create a metric alarm](zcl_aws1_cwt_actions.clas.abap#L177) (`PutMetricAlarm`)
+- [Delete alarms](zcl_aws1_cwt_actions.clas.abap#L63) (`DeleteAlarms`)
+- [Describe alarms](zcl_aws1_cwt_actions.clas.abap#L84) (`DescribeAlarms`)
+- [Disable alarm actions](zcl_aws1_cwt_actions.clas.abap#L106) (`DisableAlarmActions`)
+- [Enable alarm actions](zcl_aws1_cwt_actions.clas.abap#L130) (`EnableAlarmActions`)
+- [List metrics](zcl_aws1_cwt_actions.clas.abap#L154) (`ListMetrics`)
+
+### Scenarios
+
+Code examples that show you how to accomplish a specific task by calling multiple
+functions within the same service.
+
+- [Get started with alarms](zcl_aws1_cwt_scenario.clas.abap)
+
+
+<!--custom.examples.start-->
+<!--custom.examples.end-->
+
+## Run the examples
+
+### Instructions
+
+
+<!--custom.instructions.start-->
+<!--custom.instructions.end-->
+
+
+
+#### Get started with alarms
+
+This example shows you how to do the following:
+
+- Create an alarm.
+- Disable alarm actions.
+- Describe an alarm.
+- Delete an alarm.
+
+<!--custom.scenario_prereqs.cloudwatch_Scenario_GettingStarted.start-->
+<!--custom.scenario_prereqs.cloudwatch_Scenario_GettingStarted.end-->
+
+
+<!--custom.scenarios.cloudwatch_Scenario_GettingStarted.start-->
+<!--custom.scenarios.cloudwatch_Scenario_GettingStarted.end-->
+
+### Tests
+
+⚠ Running tests might result in charges to your AWS account.
+
+
+To find instructions for running these tests, see the [README](../../README.md#Tests)
+in the `sap-abap` folder.
+
+
+
+<!--custom.tests.start-->
+<!--custom.tests.end-->
+
+## Additional resources
+
+- [CloudWatch User Guide](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/WhatIsCloudWatch.html)
+- [CloudWatch API Reference](https://docs.aws.amazon.com/AmazonCloudWatch/latest/APIReference/Welcome.html)
+- [SDK for SAP ABAP CloudWatch reference](https://docs.aws.amazon.com/sdk-for-sap-abap/v1/api/latest/cloudwatch/index.html)
```

Comment:
This ref link is wrong--SAP ABAP uses a custom naming scheme for the services.

Reply:
Then we need to update the api ref, what is the pattern?